src/resource/solidDataset.test.ts

```diff
@@ -1583,7 +1603,9 @@ describe("saveSolidDatasetAt", () => {
       const mockFetch = jest
         .fn<typeof fetch>()
         .mockReturnValue(
-          Promise.resolve(new Response("Not found", { status: 404 }))
+          Promise.resolve(
+            new Response("Not found", { status: 404, statusText: "Not Found" })
```

Comment:
This is what I would think most of our tests to look like rather than using `mockResponse`.

That said, why is the addition of `statusText` being done?

Reply:
Ditto for the status text, and in this case we don't need the `response.url` value being mocked.